test/lib/usd/pxrUsdPreviewSurface/testPxrUsdPreviewSurfaceDraw.py

```diff
@@ -43,8 +44,6 @@ def setUp(self):
     def _WriteViewportImage(self, outputImageName, suffix):
         # Make sure the hardware renderer is available
         MAYA_RENDERER_NAME = 'mayaHardware2'
-        mayaRenderers = cmds.renderer(query=True, namesOfAvailableRenderers=True)
-        self.assertIn(MAYA_RENDERER_NAME, mayaRenderers)
 
```

Comment:
This command only seem to work in interactive. We can still continue to the ogsRender command.